src/index.js

```diff
@@ -1,6 +1,6 @@
 const BufferedStream = require('./buffered-stream')
 const generateView = require('./transpiler')
-const { Adapter, utils } = require('fractal-fork').fractal
+const { Adapter, utils } = require('@innoq/fractal-fork').fractal
```

Comment:
This seems a significant changes, which at the very least warrants more than a patch-version change? Unfortunately, I'm insufficiently contextualized to decide whether this desirable/acceptable in the first place.

Reply:
For all changes since (@innoq/)fractal-fork version 0.1.0 to 1.0.3 please read the changelog https://github.com/innoq/fractal-fork/blob/main/CHANGELOG.md

The major version change was caused by the changed package name and the end of node 14 support.

Reply:
Ah, it wasn't clear to me that this package was merely renamed.